app/assets/stylesheets/_bootstrap_overrides.scss

```diff
@@ -36,3 +40,40 @@ label.required:after {
     @extend .badge-success;
   }
 }
+
+.container {
+  padding: 0;
+}
+
+.alert {
+  padding: 0.5rem 1.25rem;
+  margin: 0;
+}
+
+h1 {
+  font-weight: bold;
+  font-size: 36px;
```

Comment:
⚠️ The problem with defining font sizes in `px` is that it doesn't allow you to increase text size in your browser, which a lot of people with poor vision do. You should instead define these in `em` so that they're relative to the base font size.